### PR TITLE
Ezp 31650 search in trash frontend

### PR DESCRIPTION
**Pull request by lucasOsti**
jira ref: https://jira.ez.no/browse/EZP-31650

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/trash/list.html.twig [hunk 1]
```diff
@@ -19,7 +20,54 @@
...
+                            <div class="ez-trash-search-form__input-wrapper">
+                                <input 
+                                    type="text"
+                                    class="form-control ez-trash-search-form__input"
+                                    data-content-type-identifiers="{{ user_content_type_identifier|join(',') }}"
+                                    value="{{ creator is not empty ? ez_content_name(creator) }}"
+                                    placeholder="{{ 'search.creator_input.placeholder'|trans|desc('Type creator\'s name') }}"
+                                    {{ creator is not empty ? 'disabled'  }}
+                                >
+                                <svg class="ez-icon ez-icon--dark ez-icon--medium ez-icon--reset">
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/trash/list.html.twig:43]: This should be inside a button and event listener should be attached to the button.

File: src/bundle/Resources/public/scss/_trash-list.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    margin: calculateRem(26px) 0 calculateRem(42px);
+
+    &__inputs-group {
+        display: flex;
+        justify-content: space-between;
+    }
+    
+    &__item {
+        display: inline-block;
+
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_trash-list.scss:11]:
```suggestion
```

**Comment by lucasOsti** [comment]
sc1
sc2

**Comment by michal-myszka** [comment]
@lucasOsti please change target branch to EZP-31650-search-in-trash

**Comment by michal-myszka** [comment]
Please continue CR in target branch

**Pull request merged**